src/BoundaryConditions.cpp

```diff
@@ -4,59 +4,55 @@ namespace Halide {
 
 namespace BoundaryConditions {
 
-Func constant_exterior(const Func &source, Expr value,
-                       const std::vector<std::pair<Expr, Expr> > &bounds) {
+Func repeat_edge(const Func &source,
```

Comment:
Did repeat_edge and constant_exterior just get re-ordered? It's a little hard to tell from this diff. (If it's just a re-order, perhaps putting them back in the same order for this PR would simplify review.)